src/components/NcActionInput/NcActionInput.vue

```diff
@@ -193,10 +194,9 @@ For the `NcSelect` component, all events will be passed through. Please see the
 							:value="value"
 							:placeholder="text"
 							:disabled="disabled"
-							:append-to-body="false"
 							:input-class="{ focusable: isFocusable }"
 							class="action-input__multi"
-							v-bind="$attrs"
+							v-bind="{ 'append-to-body': false, ...$attrs }"
```

Comment:
`kebab-case` is for templates (I'm not sure it works in Vue 3)

```suggestion
							v-bind="{ appendToBody: false, ...$attrs }"
```

Reply:
But then we also need to provide the prop as `:appendToBody`, otherwise the object spread/merge does not work.

Reply:
Oh, indeed, Vue doesn't change the case in `$attrs`, only in `$props`...

But actually it works in any way.

`{ 'append-to-body': 1, appendToBody: 2 }` and `{ appendToBody: 1, 'append-to-body': 2 }` both set value 2, using the last value for the final prop value.

So both options works the same no matter if kebab or camel case was used in template.

Reply:
So I'd call it a style preference just to keep camelCase in JS part

Reply:
I think I don't get it. I tried it with camelCase for v-bind and it didn't work. Did I do something wrong?

Reply:
Arr x2.

Vue 3 and Vue 2 works differently here... 

Vue 3 works as I described and the last one value wins.

In Vue 2 the first one wins.

Reply:
So:
- `{ 'append-to-body': false, ...$attrs }` breaks with `<NcActionInput appendToBody>`
- `{ appendToBody: false, ...$attrs }` breaks with `<NcActionInput append-to-body>`

Reply:
🥲 ?

```js
:append-to-body="$attrs.appendToBody || $attrs['append-to-body'] || false"
```

Reply:
> ```
> :append-to-body="$attrs.appendToBody || $attrs['append-to-body'] || false"
> ```

Yeah, why not. Works for both `append-to-body` and `appendToBody` in the template. For vue 3 we anyway have to / can remove it, since it works as it there.